plugins/woocommerce-blocks/assets/js/blocks/product-filters/components/upgrade.tsx

```diff
@@ -35,9 +35,9 @@ const Upgrade = ( { clientId }: { clientId: string } ) => {
 
 		replaceBlock(
 			clientId,
-			createBlock( `woocommerce/collection-filters`, {}, [
+			createBlock( `woocommerce/product-filters`, {}, [
 				createBlock(
-					`woocommerce/collection-${ filterType }`,
+					`woocommerce/product-filters-${ filterType }`,
```

Comment:
This isn't working anymore due to the block name change. For example, `filterType` is `price-filter` for the Price Filter block, but we don't have `woocommerce/product-fitlers-price-filter` block.

Reply:
Downgrade isn't working as well.

Reply:
Fixed https://github.com/woocommerce/woocommerce/pull/43363/commits/e3fe66d957d30686f0b3a80c393f747ee834d4be